.storybook/config.js

```diff
@@ -1,7 +1,7 @@
 import { configure } from '@kadira/storybook';
 import ThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
 import aphroditeInterface from 'react-with-styles-interface-aphrodite';
-import registerInterfaceWithDefaultTheme from '../scripts/utils/registerInterfaceWithDefaultTheme';
+import registerInterfaceWithDefaultTheme from '../lib/utils/registerInterfaceWithDefaultTheme';
```

Comment:
I don't think we want this update. Let's pull from src here. Otherwise we'll have to build all the js every time we run storybook